src/core/core_utils.js

```diff
@@ -205,7 +205,23 @@ function escapePDFName(str) {
   let start = 0;
   for (let i = 0, ii = str.length; i < ii; i++) {
     const char = str.charCodeAt(i);
-    if (char < 0x21 || char > 0x7e || char === 0x23) {
+    // whites or delimiter aren't regular chars
+    // so escape them
```

Comment:
"whites", did you actually mean "whitespace" here?

Also, please improve the formatting of this comment, since the line-break is just overall confusing (i.e. is this intended to be two sentences, or not):
```suggestion
    // Whitespace or delimiters aren't regular chars, so escape them.
```